src/models/StuntSheetDot.ts

```diff
@@ -37,6 +38,8 @@ export default class StuntSheetDot extends Serializable<StuntSheetDot> {
 
   cachedFlow: FlowBeat[] | null = null;
 
+  warnings: Warning[] = [];
```

Comment:
Right now warnings are scattered across the program, in the show, stunt sheets, or dots. Does it make sense to consolidate into one place? Or is it better to keep them associated with objects like this? (e.g. Are there cases where it might be useful to associate a warning with more than one sheet or more than one dot? What is the recommendation for deciding which object should claim ownership of a warning in this case?)

Reply:
That's a good question. The reasoning behind why I didn't make a single `warnings` array was for cacheing. I didn't want to have to recalculate all the warnings if one gets fixed. By having them spread out among the different objects, warnings only get recalculated if they need to be. 

Currently one of the warnings that we have multiple ownerships for is the warning that comes out when there are different numbers of dots between two stunt sheets. Because this happens between two separate stunt sheets, I thought that instead of being associated with either sheet itself it's instead associated with the show as a whole (one level up). A similar thing happens with dots: when two dots are overlapping the new warning is added to the sheet's array and not either of the dots.